Library/Formula/docker.rb

```diff
@@ -2,8 +2,8 @@ class Docker < Formula
   desc "Pack, ship and run any application as a lightweight container"
   homepage "https://www.docker.com/"
   url "https://github.com/docker/docker.git",
-      :tag => "v1.9.1",
-      :revision => "a34a1d598c6096ed8b5ce5219e77d68e5cd85462"
+      :tag => "v1.10.0",
+      :revision => "590d5108bbdaabb05af590f76c9757daceb6d02e"
   revision 1
```

Comment:
@outcoldman Please try to be patient. Sometimes maintainers are busy elsewhere; nobody's going to die because it wasn't merged for 12 hours :wink:.